scipy/stats/tests/test_resampling.py

```diff
@@ -332,14 +332,15 @@ def test_bootstrap_against_itself_1samp(method, expected):
 
     # Do the same thing 2000 times. (The code is fully vectorized.)
     n_replications = 2000
-    data = dist.rvs(size=(n_replications, n))
+    data = dist.rvs(size=(n_replications, n), random_state=rng)
     res = bootstrap((data,),
                     statistic=np.mean,
                     confidence_level=confidence_level,
                     n_resamples=n_resamples,
                     batch=50,
                     method=method,
-                    axis=-1)
+                    axis=-1,
+                    random_state=rng)
```

Comment:
Due to [SPEC 7](https://scientific-python.org/specs/spec-0007/), Please use `rng` instead of `random_state` where possible. (That's pretty much everwhere that accepts `random_state` or `seed` right now *except* the old distribution infrastructure, like the call to `dist.rvs` above.

On a related note, please use of `default_rng` instead of `RandomState` to generate instances that get passed into functions when there is a choice. Internally, these functions will (in the future) use `default_rng` to normalize the `rng` argument, and until NumPy 2.2, `default_rng` will not accept an instance of `RandomState` as an argument.

Reply:
Renamed from `random_state` to `rng`, resolving.